Automattic/wp-calypso
# Add contextual links for expired and recently registered domains

#### Changes proposed in this Pull Request

* The "Learn more" links for expired or recently registered domains should point to more specific support documents

#### Testing instructions

* Open a recently registered or expired domain in the new domain status screen and click the "Learn more" link - verify it's pointing to a contextually relevant support document.
